### PR TITLE
Change YoroiNFT and TokenInfo data structure

### PR DESCRIPTION
**Pull request by michalszorad**
Resolves YOMO-458

### CODE REVIEW

File: src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx [hunk 1]
```diff
@@ -91,9 +91,13 @@ const SelectableAssetItem = ({tokenInfo, disabled, wallet}: SelectableAssetItemP
     tokenSelectedChanged(tokenInfo.id)
 
     // if the balance is atomic there is no need to edit the amount
-    if (Quantities.isAtomic(spendable, tokenInfo.decimals)) {
+    if (tokenInfo.kind === 'ft' && Quantities.isAtomic(spendable, tokenInfo.metadata.decimals)) {
       amountChanged(spendable)
       navigation.navigate('send-list-amounts-to-send')
+    }
+    if (tokenInfo.kind === 'nft') {
+      amountChanged('1')
```
Comment (banklesss) [inline src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx]: Sharing what @stackchain commented here https://github.com/Emurgo/yoroi-mobile/pull/2440#discussion_r1171975709

This is my solution: https://github.com/Emurgo/yoroi-mobile/blob/2ecdba49134b40271bfb5a54f1afd94713cc9f0e/src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx#L97

File: src/yoroi-wallets/storage/storage.test.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import AsyncStorage from '@react-native-async-storage/async-storage'
 import {parseSafe} from '../utils/parsing'
 import {storage as yoroiStorage} from '.'
 
-describe('prefixed storage', () => {
+describe.only('prefixed storage', () => {
```
Comment (banklesss) [inline src/yoroi-wallets/storage/storage.test.ts]: Should `only` be removed?

**Comment by michalszorad** [comment]
Rebase coming soon

**Review by banklesss — APPROVED** [review]

**Comment by michalszorad** [comment]
Closing in favour of https://github.com/Emurgo/yoroi/pull/2461

**Pull request closed without merging**